src/uiTest/resources/project.rhino.js

```diff
@@ -297,16 +301,17 @@ function Project(underlyingProject) {
   const getIdeFrame = function() {
     return getComponentByClassAndText(
         /* className */ 'com.intellij.openapi.wm.impl.IdeFrameImpl',
-        /* text */ 'machete-sandbox-worktree',
-        /* textCmp */function (text, component) {
-          return text.equals(component.getTitle());
-        }
+        // Using "includes" as in UITestSuite the project title is "machete-sandbox-worktree"
+        // but in UIScenarioSuite it's "machete-sandbox"
+        /* predicate */ component => 'machete-sandbox-worktree'.includes(component.getTitle())
```

Comment:
Hmmm and if the title is empty (as used to be for the other `IdeFrameImpl`s)?
Shouldn't it be:
```suggestion
        /* predicate */ component => component.getTitle().contains('machete-sandbox')
```